incubator/aci/create-k8-connector/script.md

```diff
@@ -114,30 +121,10 @@ spec:
 kubectl create -f examples/nginx-pod.yaml 
 ```
 
-```
-kubectl get pods -o wide
-```
-
-Since we need to ensure our Public IPs have been assigned before
-proceeding, and because we need the IP number later we'll run a loop
-to grab the IP once assinged. This is a little cumbersome but great if
-you want to script things. If you are doing this manually you can use
-`kubectl get service --wait` to display changes as they happen.
-
-```
-NGINX_IP=""
-while [ -z $NGINX_IP ]; do sleep 10; NGINX_IP=$(kubectl get service vamp -o jsonpath="{.status.loadBalancer.ingress[*].ip}"); done
-```
-
-Now we have our IP:
-
-```
-echo $NGINX_IP
-```
-
-Take a look...
+This command grabs the pods running on the cluster and we can see that 
+the Nginx pod is running on the connector which means it's running in as Azure Container Instance.
 
 ```
-xdg-open $NGINX
+kubectl get pods -o wide
```

Comment:
Nope this is not enough for a demo. This says it's running but it doesn't prove it is.

Reply:
I don't know if it worked or not - I'll try again though